benches/byteorder_1_2_6/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "byteorder_1_2_6"
 version = "0.1.0"
-authors = ['Thom wiggers <thom@thomwiggers.nl>']
+license = "Unlicense/MIT"
```

Comment:
Technically, the changes made to adopt benchmarks into lolbench could be covered by a different license from the parent project (if the parent license allows that).  For byteorder the changes are probably not material enough to warrant that, that's not necessarily true in general.

Some licenses also require attribution, so I would suggest that we add [some header text](https://github.com/anp/lolbench/blob/master/benches/byteorder_1_2_6/src/lib.rs#L1-L8) to modules that include copied source code.

Reply:
Yeah, licensing the lolbench specific changes differently is totally legit, but that sounds like a lot of work to explain to someone. I agree that including header text on copied source files is a good idea!